apiml-common/src/main/java/com/ca/mfaas/product/logging/ApimlLogInjector.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+package com.ca.mfaas.product.logging;
+
+import com.ca.mfaas.message.core.MessageService;
+import com.ca.mfaas.message.log.ApimlLogger;
+import com.ca.mfaas.product.logging.annotations.InjectApimlLogger;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.config.BeanPostProcessor;
+import org.springframework.context.ApplicationContext;
+import org.springframework.stereotype.Component;
+import org.springframework.util.ReflectionUtils;
+
+/**
+ * This class plays as a injector for {@link ApimlLogger} in spring environment.
+ * It detects if class have field with {@link InjectApimlLogger} annotation,
```

Comment:
"Detects if the class has a field (input field?) with the {@link InjectApimlLogger} annotation"